manifests/istio-control/istio-discovery/values.yaml

```diff
@@ -174,9 +174,12 @@ telemetry:
     # For Null VM case now. If enabled, will set disableMixerHttpReports to true and not define mixerReportServer
     # also enable metadata exchange and stats filter.
     enabled: true
+    metadataExchange:
```

Comment:
For clarity fix the comments above.

Reply:
Bike shed: “useWasm” ?

Reply:
This PR is to track e2e test of the Wasm filter and the option will be added in https://github.com/istio/istio/pull/21776. I will address your comment there.

Regarding to the name.. both names make sense to me so I don't have much preference :) `wasmEnabled` is more consistent with other name in the telemetry block, so I'll stick with it.